source/about-mongosync.txt

```diff
@@ -157,6 +157,31 @@ If the reverse sync is successful, ``mongosync`` enters the ``RUNNING`` state.
 The synchronization continues in the reverse direction from the original 
 sync job. 
 
+.. _mongosync-considerations:
+
+Considerations for Continuous Sync
+----------------------------------
+
+Consider the following when using ``mongosync`` for continuous synchronization:
+
+- For the destination cluster to have an up to date, consistent view of the 
+  source cluster’s data, ``mongosync`` must :ref:`commit <c2c-api-commit>` . 
+  If the source cluster shuts down before ``mongosync`` can commit, such as in 
+  a disaster scenario, the destination cluster might not have a consistent 
+  snapshot of the source data, and transactions might not appear atomically on 
+  the destination cluster.
+  
+  - For any continuous synchronization use cases with ``mongosync``, ensure that 
+    ``mongosync`` commits before cutting over from the source to the 
+    destination.
+
+- When used with sharded clusters, you must pause :ref:`balancers 
+  <sharding-balancing>` on both the source and the destination for the lifetime 
+  of the sync.
```

Comment:
feedback from Ritesh: add "until commit" to "lifetime of the sync"

Reply:
CC @rohshar

Reply:
fine with me